nemo/collections/asr/models/hybrid_rnnt_ctc_models.py

```diff
@@ -441,6 +435,17 @@ def training_step(self, batch, batch_nb):
                 self.ctc_wer.reset()
                 tensorboard_logs.update({'training_batch_wer_ctc': ctc_wer})
 
+        # note that we want to apply interctc independent of whether main ctc
```

Comment:
How would be the final loss when ctc_loss_weight=0.3 and interctc.loss_weights=[0.1]? Something like this?
`final_loss=0.9(0.7(rnnt_loss)+3.0(ctc_loss_final_layer))+0.1(ctc_loss_middle_layer)`

Reply:
yes, that's correct. I will add an explicit formula here